tests/catch/src/main/graphql/schema.graphqls

```diff
@@ -0,0 +1,26 @@
+#extend schema @link(url: "https://specs.apollo.dev/nullability/v0.1")
+
+extend schema @catch(to: THROW)
```

Comment:
I'm not sure I understand how this works

Reply:
That's the default value for `@catch` if nothing is specified in the operation:

```graphq
# no codegen change, current situation
extend schema @catch(if: false)
# throw on every error by default => data and exception are now exclusive
extend schema @catch(to: THROW)
# model every field as a result (probably a bit too much 😅)
extend schema @catch(to: RESULT)
```

I should probably write some docs

Reply:
Separate doc PR: https://github.com/apollographql/apollo-kotlin/pull/5407